docs/CodeDevelopment/requirements.md

```diff
@@ -32,21 +32,50 @@
         }
     ```
 
-## Uefi Package
+## Repositories
 
-## UEFI Components
+[Repo Philosophy](overview.md#repo-philosophy)
 
-1. All new modules must be listed in their containing package DSC in the components section
+- Repositories should have a clear owner
+- Repositories should have a clear target
+  - Cross platform/single platform
+  - Cross architecture/single architecture
+  - Cross organization/single organization
+- Repositories should have a consistence license (open source or proprietary)
```

Comment:
consistence?? constant?

Reply:
Can we also add something about IP restrictions.

Reply:
Fixed in latest.